tests/kafka.test.js

```diff
@@ -0,0 +1,127 @@
+'use strict'
+
+const tap = require('tap')
+const { Kafka, logLevel: kafkaJsLogLevels } = require('kafkajs')
+const customLogger = require('../index')
+
+tap.test('Kafka client with custom logger', t => {
```

Comment:
We could add a test with the schema of the output log, to verify it follows the Mia guidelines, as made in [other libs](https://github.com/mia-platform/lc39/blob/master/tests/log.schema.json)

Reply:
How is a log example?